docs/DeveloperGuide.adoc

```diff
@@ -767,6 +785,255 @@ Use case ends.
 +
 Use case resumes at step 2.
 
+[discrete]
+=== Use case: Add person details
+
+*MSS*
+
+1.  User requests to create a contact with more details including birthday, home call number and so on
+2.  User types in these details
```

Comment:
a use case should show back and forth interaction  between user and the system. Thus, it's incorrect to have consecutive user or system interactions.